//package com.sincetimes.redwar.game.comnunicate.protocol;


message RceObtainNeighborsList
{
	
}

